ironfish/src/rpc/routes/accounts/getTransaction.ts

```diff
@@ -3,26 +3,25 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 import * as yup from 'yup'
 import { ApiNamespace, router } from '../router'
-import { getAccount, getTransactionNotes, getTransactionStatus } from './utils'
+import { serializeRpcAccountDecryptedNote, serializeRpcAccountTransaction } from './types'
+import { getAccount, getTransactionStatus } from './utils'
 
 export type GetAccountTransactionRequest = { account?: string; hash: string }
 
 export type GetAccountTransactionResponse = {
```

Comment:
the `GetAccountTransactionResponse` and `GetAccountTransactionRequest` are updated with non backward compatible change, is this okay

Reply:
I think so, since the new command uses it. We'll have to focus on backwards compatibility at some point.